pydicom/values.py

```diff
@@ -192,26 +262,32 @@ def convert_PN(byte_string, is_little_endian, struct_format=None, encoding=None)
 
     # We would like to return string literals
     if not in_py2:
-        valtype = lambda x: PersonName(x, *args).decode()
+        valtype = lambda x: PersonName(x, *args).decode()  # nopep8
     else:
-        valtype = lambda x: PersonName(x, *args)
+        valtype = lambda x: PersonName(x, *args)  # nopep8
 
```

Comment:
Ah ok, mixed up python 2 and 3.
This works:
``` python
def convert_PN(byte_string, is_little_endian, struct_format=None, encoding=None):
    """Read and return string(s) as PersonName instance(s)"""

    def get_valtype(x):
        if not in_py2:
            if encoding:
                return PersonName(x, encoding).decode()
            return PersonName(x).decode()
        return PersonName(x)

    # XXX - We have to replicate MultiString functionality here because we can't decode
    # easily here since that is performed in PersonNameUnicode
    if byte_string and (byte_string.endswith(b' ') or byte_string.endswith(b'\x00')):
        byte_string = byte_string[:-1]

    splitup = byte_string.split(b"\\")

    if len(splitup) == 1:
        return get_valtype(splitup[0])
    else:
        return MultiValue(get_valtype, splitup)


```